talk/tools/webtools.tex

```diff
@@ -0,0 +1,70 @@
+\subsection[web]{Web tools}
+
+\begin{frame}
+  \frametitle{Godbolt / Compiler Explorer }
+  \begin{block}{Concept}
+    An online generic compiler with immediate feedback.
+    Allows to :
+    \begin{itemize}
+    \item compile online any code against any version of any compiler
+    \item inspect the assembly generated
+    \item use external libraries (over 50 available !)
+    \item run the code produced
+    \item use tools, e.g. ldd, include-what-you-use, ...
+    \item share small pieces of code via permanent short links
```

Comment:
I wouldn't use "Allow to", see e.g. [english stackexchange](https://english.stackexchange.com/questions/60271/grammatical-complements-for-allow/60285#60285).

We could use:
```suggestion
    Allows:
    \begin{itemize}
    \item compiling any code online against any version of any compiler
    \item inspecting the assembly generated
    \item use of external libraries (over 50 available !)
    \item running the code produced
    \item using tools, e.g.\ ldd, include-what-you-use, ...
    \item sharing small pieces of code via permanent short links
```